pom.xml

```diff
@@ -32,7 +32,7 @@
 			</snapshots>
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
+			<url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-snapshots</url>
```

Comment:
You have eagle eyes, nice catch 👍

Reply:
However it should not have been related to the issue, where Github Actions could not resolve the dependency for the data-model-lib 🤦 .

Reply:
Works on my machine ... :O